Apps/phdigitalguardianarc/readme.html

```diff
@@ -0,0 +1,21 @@
+<p>
+  This App is an Ingestion source. In the Phantom documentation, in the
+  <a href="../admin/">Administration Manual</a> under the
+  <a href="../admin/sources">Data Sources</a> section, you will find an
+  explanation of how Ingest Apps works and how information is extracted from the
+  ingested data. There is a general explanation in Overview, and some
+  individuals Apps have their own sections.
+</p>
```

Comment:
the referenced section of the documentation no longer exists. I removed the same blurb from another app in PAPP-10405 for that reason

Reply:
Removed doc reference link as mentioned in PAPP-10405